tests/test_crawler_container.py

```diff
@@ -96,7 +97,9 @@ def testCrawlContainer1(self):
 		'partition_strategy': {
 		    'args': {
 			'process_id': 0,
-			'num_processes': 1
+			'num_processes': 1,
+			'eventQ':multiprocessing.JoinableQueue(),
```

Comment:
This is another reason for why i would prefer to not have this in here. All the other options are text, and most of them are optional. If we don't pass these new options, the crawler crashes, that shouldn't happen.